Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/ShaderStages.h

```diff
@@ -32,6 +32,10 @@ namespace AZ::RHI
         //! a vertex. On certain platforms like Metal, this stage may occur after tessellation.
         Vertex = 0,
 
+        //! This virtual stage contains shader stages that expand an input assembly stream and manipulate
+        //! a vertex.
+        Geometry,
+
         //! This virtual stage contains platform-specific stages for hardware tessellation. The specifics
```

Comment:
Consider removing Tessellation. It is left over code from before. This may allow DDGI's precompiled shaders to still function as you are replacing an enum instead of adding a new one. Re-creating the precompiled shaders is a bit of a pain and you will need help from someone internal to Amazon with that and that may take some time.

Reply:
I have removed all the code related to tessellation